README.rst

```diff
@@ -163,35 +163,58 @@ with:
 
 	$ grunt watch
 
-It is also possible to develop figure in docker without installing anything locally.
-The Docker image is built on top of ``openmicroscopy/omero-web-standalone:latest``, so you will have a fully functional
-omero-web environment while developing ``omero-figure``. 
-First build the Docker image:
+
+Local install
+*************
+
+You need an `omero-web` environment. You can either install it locally or use Docker (see below).
+To install locally, we recommend that you use `conda` to install `omero-py` as described 
+at https://github.com/ome/omero-py and then perform a developer install of `omero-figure`
+which will include `omero-web`:
 
 ::
 
-   $ docker build -t figure-devel .
+    $ cd omero-figure
+    $ pip install -e .
+
+See instructions at https://omero.readthedocs.io/en/latest/developers/Web/Deployment.html
+for running the development server.
 
+Using Docker
+************
+
+It is also possible to develop figure in docker without installing anything locally.
```

Comment:
This is obviously not true. After consultaion, I understand that you have to install both npm and grunt.

Reply:
Tweaked this in f35ae2e